README.md

```diff
@@ -1,3 +1,55 @@
+# Cov-Clear web client
+
+The cov clear backend is a platform designed to securely manage:
+
+- The administration of mass community testing for covid-19.
+- The ability to securely share test results with others, acting as an "immunity passport".
+- The ability to anlyse aggregated and anonymised test result data.  Helping to establish:
+  - the efficacy of different tests.
+  - the persistence of antibodies over time.
+  - patient reinfection rates.
+  - variances by demographic segments
+
+### User interaction
+
+For more information on how the platform can be used, please refer to the documentation for the corresponding [https://github.com/cov-clear/web/blob/master/README.md](web application).
+
+### Authentication
+
+Users register and log in to the system through 'magic-links', these are one-time links that are sent into user's email inbox.  The link contains an authCode, that can be exchanged for a secure token used for future requests.  The link is one-time, as it expires on first usage.  To login again, a new magic link, containing a new auth code, must be requested.  As such, the system does not contain any passwords, and can only be accessed via the users email inbox.
+
+### Data capture
+
+The application asks users to submit both demographic (date of birth & sex) & location (full address) information, to enable demographic segmentation of results.  Address information may be used to verify a petients identity, and at later stages to track outbreaks via test results.  Data is only communicated over secure SSL connections.
+
+### Data access & sharing results
+
+The system is secured such that patients control their own data. A patient may issue a unique, short-lived, sharing code that enables another user to securely view and verify those results. Sharing codes have a life span of only 60 seconds to prevent their leakage and to prevent any brute force attempt to crack codes.
+
+A sharing codes will typically be converted into a QR code which can be captured by another user. On capture, the capturer is granted an 'access pass' giving them permissions to view the sharers test result data (currently for one hour).  There is no other way to lookup or view a patients data, ensuring the patient controls who can access their data.
+
+If the capturing user has sufficient permission (e.g. they are a registered doctor), they may add new results on behalf of a patient.
```

Comment:
Might want to specify that this is also only if they have an access pass currently